README.md

```diff
@@ -76,7 +76,17 @@ docker run --gpus all \
     --env "HF_TOKEN=<secret>" \
     --ipc=host \
     lmsysorg/sglang:latest \
-    python3 -m sglang.launch_server --model-path meta-llama/Meta-Llama-3-8B-Instruct --host 0.0.0.0 --port 30000
+    --model-path meta-llama/Meta-Llama-3-8B-Instruct --host 0.0.0.0 --port 30000
```

Comment:
For Docker startup, if you have k8s configurations, please create a separate document. I aim to keep the README as straightforward and user-friendly as possible.
ref https://github.com/InternLM/lmdeploy/tree/main/k8s
And it's best to verify whether this configuration can run.